app/src/app/home/pages/element/element.model.ts

```diff
@@ -46,7 +46,11 @@ export class ElementPageMetric extends PageListItem<
     if (k.score && k.score === 1) {
       k.dstValue = k.srcValue;
     }
-    const val = new Metric(k.name, +k.srcValue || null, +k.dstValue || null);
+    const parse = (v: string | undefined) => {
+      const x = parseFloat(v);
+      return isNaN(x) ? null : x;
+    };
+    const val = new Metric(k.name, parse(k.srcValue), parse(k.dstValue));
```

Comment:
Unrelated bugfix: the previous logic did not correctly handle the case when the value is "0".